src/v2/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -157,22 +158,33 @@ export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
     const selectedBid = Number(values.selectedBid)
 
     if (requiresPaymentInformation) {
+      const stripeAddress = toStripeAddress(values.address)
```

Comment:
I guess there's no risk in these 3 expressions throwing an exception – is that why you moved out of the `try`?

Reply:
There is a risk of an error there as well, but a lot less likely since it doesn't involve any network calls or I/O. When there is a bug in e.g. `toStripeAddress` then [the generic `ErrorBoundary` component](https://github.com/artsy/force/blob/8bf9635a32558320e3ace910d651c973050d8bdc/src/v2/Artsy/Router/ErrorBoundary.tsx#L19) should will that and present an appropriate message in a modal.